src/dbg/debugger_commands.cpp

```diff
@@ -142,6 +142,11 @@ CMDRESULT cbDebugRun(int argc, char* argv[])
     // Don't "run" twice if the program is already running
     if(dbgisrunning())
         return STATUS_ERROR;
+    if(!ThreaderIsAnyThreadActive())

```

Comment:
Does this actually make sense to do? Run does not do anything about threads, it just releases the run locks and makes sure `ContinueDebugEvent` is called...